README.md

```diff
@@ -10,11 +10,12 @@ Moneo orchestrates metric collection (DCGMI + Prometheus DB) and visualization (
 
 There three categories of metrics that Moneo monitors:
 1.	Device Counters
-	- Compute/Memory Utilization
-	- SM and Memory Clock frequency
-	- Temperature
-	- Power
+    - Compute/Memory Utilization
+    - SM and Memory Clock frequency
+    - Temperature
+    - Power
     - ECC Counts
+    - GPU Throttling
 2.	Profiling Counters
```

Comment:
Are we disabling these by default?

Reply:
We discussed disabling the profiling counters which I will be introducing in my next PR. I don't think we discussed making throttling optional. If we go down this path, I would like to introduce it in my next PR because I am doing similar work like this already in that branch.

Reply:
thanks @rafsalas19, yes i was referring to profiling counters only. it will be good to enable throttling by default.